Zotero/Scenes/Detail/PDF/PDFCoordinator.swift

```diff
@@ -30,6 +30,17 @@ protocol PdfReaderCoordinatorDelegate: AnyObject {
 }
 
 protocol PdfAnnotationsCoordinatorDelegate: AnyObject {
+    func shareAnnotationMenu(
```

Comment:
Rename to `createShareAnnotationMenu` since it returns the menu? I wouldn't really notice this by itself, but below there is `func  shareAnnotationImage(...)` and it actually shares the image, doesn't return an image to share, so it caught my eye.